testing/run-scripts/image_make.sh

```diff
@@ -1,21 +1,85 @@
 #!/bin/bash
 
-# image_make.sh browser version
-# builds a docker image called uproxy/browser-version
+set -e
+
+GIT=false
+BRANCH=
+PREBUILT=false
+
+function usage () {
+  echo "$0 [-g] [-b branch] [-p] [-h] browser version"
+  echo "  -g: pull code from git (conflicts with -p)"
+  echo "  -b: git branch to pull (default: HEAD's referant)"
+  echo "  -p: use a pre-built uproxy-lib (conflicts with -g)"
+  echo "  -h, -?: this help message"
+  echo
+  echo "Without -g or -p the latest uproxy-lib NPM will be run."
+  exit 1;
+}
+
+while getopts gb:ph? opt; do
+  case $opt in
+    g) GIT=true ;;
+    b) BRANCH="-b $OPTARG" ;;
+    p) PREBUILT=true ;;
+    *) usage ;;
+  esac
+done
+shift $((OPTIND-1))
+
+if [ "$GIT" = true ] && [ "$PREBUILT" = true ]
```

Comment:
May also be worth checking if -b is specified but -g isn't (doesn't cause an error but the -b is useless in that case).